src/activitylog/activitylog.html

```diff
@@ -46,6 +46,22 @@ <h2>Activity Logs</h2>
     <template id="logTableTemplate">
       <link rel="stylesheet" href="/lib/web-component/log-view/log-view.css" />
       <div class="activity-log-container">
+        <div class="timestamp-filter-container" hidden>
+          <div class="label">
+            Timestamp Filter Applied
+            <div id="clearBtn" title="Clear timestamp filter"></div>
+          </div>
+          <div class="timestamp-filter-options">
+            <div class="start-time-container">
+              <div class="title">Start Time:</div>
+              <div id="startTimeLabel">From beginning</div>
+            </div>
+            <div class="stop-time-container">
+              <div class="title">Stop Time:</div>
+              <div id="stopTimeLabel">Up to End</div>
+            </div>
+          </div>
+        </div>
```

Comment:
It feels like this should be with the other filter options:
- defined to be outside of the log view table, part of the filter wrapper defined between lines 30-41
- defined as its own webcomponent, in this version of the patch it seems that the webcomponent that manage the table view is also managing this part of the DOM and it does feel a bit outside of the responsabilities of the `LogView` webcomponent

Reply:
done :-)